src/chrome/content/rules/mastername.ru.xml

```diff
@@ -0,0 +1,23 @@
+<!--
+mastername.ru/: (52, 'Empty reply from server')
+www.mastername.ru/: (52, 'Empty reply from server')
```

Comment:
I was not able to reproduce. If this is a Travis related error, you may add the ruleset to the whitelist.